packages/core/src/types.ts

```diff
@@ -230,6 +230,9 @@ export type GlobalMockOptions = {
   useExamples?: boolean;
   // This is used to set the delay to your own custom value
   delay?: number | (() => number);
+  // This is used to execute functions that are passed to the 'delay' argument
+  // at runtime rather than build time.
+  delayCalcRuntime?: boolean;
```

Comment:
Could you please add a new regression test case to the following file:

https://github.com/anymaniax/orval/blob/master/tests/configs/default.config.ts

Reply:
Done!